Downloader-Plugins are now in one repo

sean-morris opened the pull request:
We brought all the download-plugins, the common utility file as well as
the tests for the utility file into one repository.

sean-morris committed:
Downloader-Plugins are now in one repo

We brought all the download-plugins, the common utility file as well as
the tests for the utility file into one repository.

consideRatio commented on src/nbgitpuller_downloader_generic_web/generic_web_downloader.py:
```diff
@@ -0,0 +1,21 @@
+
+    return result_handle
+
+    return { "repo": helper_args["repo"], }
```
2x return statements

sean-morris replied:
that one hurts good catch

consideRatio reviewed (approved):
Nice lets go for a merge and iterate from there if needed.

Go ahead and self-merge this after fixing the 2x return statements i commented about.

sean-morris committed:
Fixed Second Return

Somehow I spaced out a second return statement in
generic_web_downloader.py. It is gone.
